### PR TITLE
Feat/#42/2fa verify email action

### PR DESCRIPTION
**Pull request by daryafurman**

### CODE REVIEW

File: src/util/encryption/encrypt-passwords.ts
Comment (SimonSchoetz) [inline src/util/encryption/encrypt-passwords.ts:1]: Please write tests to make sure all the edge cases you thought about are not lost if somebody is refactoring this in the future :)

**Review by SimonSchoetz — CHANGES_REQUESTED** [review]

File: src/util/encryption/encrypt-password.test.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { encryptPassword } from './encrypt-passwords';
+
+describe('encryptPassword', () => {
+  it('Should only accept strings', () => {
+    const noString = 123 as unknown as string;
+    expect(encryptPassword(noString)).rejects.toThrow();
```
Comment (Femlives) [inline src/util/encryption/encrypt-password.test.ts:6]: you sure these tests do what you think they do? `encryptPassword` is an async function but I see no async/await declarations

**Review by SimonSchoetz — APPROVED** [review]
